### PR TITLE
Add hamburger, change position and size logIn and signIn buttons

### PR DESCRIPTION
**Pull request by Tkach-Illia**

### CODE REVIEW

File: front-end/src/components/UI/Hamburger/Hamburger.jsx [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+import Lines from "./Lines/Lines";
+import NavBar from "./NavBar/NavBar";
+
+const Hamburger = () =>
+{
+    const [visibleNavBar, setVisibleNavBar] = useState(false)
+
+    if(visibleNavBar){
+
+    }
```
Comment (azazak123) [inline front-end/src/components/UI/Hamburger/Hamburger.jsx]: I think we can remove this

File: front-end/src/components/UI/Hamburger/Hamburger.jsx [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 
-const Hamburger = () => {
     return (
         <div>
-            <hr className={styles.hamburger} style={{marginTop: '1%'}}/>
-            <hr className={styles.hamburger} style={{marginTop: '2%'}}/>
-            <hr className={styles.hamburger}/>
+            <Lines visible={visibleNavBar} callback={setVisibleNavBar}/>
```
Comment (azazak123) [inline front-end/src/components/UI/Hamburger/Hamburger.jsx]: Does make sense to remove `visible={visibleNavBar}` and use `callback={() => setVisibleNavBar(!visibleNavBar)}` ?

File: front-end/src/components/UI/Hamburger/NavBar/NavButton/NavButton.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cl from './NavButton.module.css'
+import {useNavigate} from "react-router-dom";
+
+const NavButton = ({text, path="/", callback}) => {
```
Comment (azazak123) [inline front-end/src/components/UI/Hamburger/NavBar/NavButton/NavButton.jsx:5]: Could we make something like `RouterButton`, which we can use not only in navbar?

File: front-end/src/components/UI/Hamburger/NavBar/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cl from './NavBar.module.css'
+import NavButton from './NavButton/NavButton';
+
+const NavBar = ({visible, callback}) => {
```
Comment (azazak123) [inline front-end/src/components/UI/Hamburger/NavBar/NavBar.jsx:5]: We send this callback through three components, could we use state management library to simplify this?

File: front-end/src/components/UI/Hamburger/NavBar/NavButton/NavButton.jsx (hunk 3 above)
Comment (Tkach-Illia) [reply front-end/src/components/UI/Hamburger/NavBar/NavButton/NavButton.jsx:5]: This makes no sense. Navbar should be the only way to navigate

File: front-end/src/components/UI/Hamburger/NavBar/NavBar.jsx (hunk 4 above)
Comment (Tkach-Illia) [reply front-end/src/components/UI/Hamburger/NavBar/NavBar.jsx:5]: No, maybe in next projects

**Review by azazak123 — APPROVED** [review]

**Pull request merged**